index.js

```diff
@@ -0,0 +1,151 @@
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+}
+
+const loadTrips = () => {
+  const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+  const Trips = $('#trip-list');
+  Trips.empty();
+
+  reportStatus('Trips Loading...')
+
+  axios.get(URL)
+  .then((response) => {
+    Trips.append(`<h1>All Trips</h1>`);
+    response.data.forEach((trip) => {
+      Trips.append(`<li id='${trip.id}'>${trip.name}</li>`);
+    });
+    reportStatus('')
+  })
+
+  .catch((error) => {
+
+    reportStatus(`Error: ${error.message}`)
+  });
+};
+
+const loadTripsbyContinent = (continent) => {
+  const continentURL = 'https://ada-backtrek-api.herokuapp.com/trips/continent?query=';
+
+  const Trips = $('#trip-list');
+  Trips.empty();
+
+  reportStatus('Trips Loading...')
+
+  axios.get(`${continentURL}${continent}`)
+  .then((response) => {
+    Trips.append(`<h1>Trips in ${continent}</h1>`);
+    response.data.forEach((trip) => {
+      Trips.append(`<li id='${trip.id}'>${trip.name}</li>`);
+    });
+    reportStatus('')
+  })
+
+  .catch((error) => {
+
+    reportStatus(`Error: ${error.message}`)
+  });
+};
+
+
+const TripsbyBudget = () => {
+  const budgeturl = 'https://ada-backtrek-api.herokuapp.com/trips/budget?query=';
+
+  let budget = $('.budget input').val()
+
+  reportStatus('Trips Loading...')
+
+  axios.get(`${budgeturl}${budget}`)
+  .then((response) => {
+    $('#trip-list').append(`<h1>Trips</h1>`);
+    response.data.forEach((trip) => {
+      $('#trip-list').append(`<li id='${trip.id}'>${trip.name}</li>`);
+    });
+    reportStatus('')
+  })
+  .catch((error) => {
+
+    reportStatus(`Error: ${error.message}`)
+  });
+}
+
+const loadOneTrip = (id) => {
+  const TripURL = 'https://ada-backtrek-api.herokuapp.com/trips/'
+
+  axios.get(`${TripURL}${id}`)
+  .then((response) => {
+    $('#trip').addClass(id)
+    $('#trip').append(`<h1>Trip Details</h1>`);
+    $('#trip').append(`<h3> <span>${response.data.name}</span></h3>`);
+    $('#trip').append(`<p>Category: ${response.data.category}</p>`);
+    $('#trip').append(`<p># of Weeks: ${response.data.weeks}</p>`);
+    $('#trip').append(`<p>Cost: $${response.data.cost.toFixed(2)}</p>`);
+    $('#trip').append(`<p>About: ${response.data.about}</p>`);
+    LoadReservationForm(id)
+  })
+  .catch((error) => {
+    reportStatus(`Error: ${error.message}`)
+  });
+};
+
+const LoadReservationForm = (id) => {
+  $('#reservations').prepend(`<h1>Reserve This Trip</h1>`);
```

Comment:
Maybe I don't understand what's going on, but why isn't this part just hardcoded in?